.rubocop.yml

```diff
@@ -0,0 +1,37 @@
+require: 
+  - rubocop-rails
+
+AllCops:
+  NewCops: enable
+  Exclude:
+    - 'db/**/*'
+    - 'bin/**/*'
+    - 'Gemfile'
+    - 'Rakefile'
+    - 'spec/spec_helper.rb'
+    - 'node_modules/**/*'
+    - 'tmp/*'
+    - 'config/initializers/**/*'
+    - 'lib/tasks/*'
+    - 'Guardfile'
+
+Layout/LineLength:
+  Max: 100
```

Comment:
I personally don't mind the default line length of `120` but am very open to other opinions on that.

Reply:
That would clear up a bunch of those offenses, I am open to changing this.